packages/bpk-react-utils/src/Portal.js

```diff
@@ -243,11 +247,16 @@ class Portal extends Component {
     const missesExpectedTarget = this.props.target && !this.getTargetElement();
 
     if (this.portalElement && !missesExpectedTarget) {
-      render(this.props.children, this.portalElement, () => {
-        if (this.props.isOpen) {
-          this.props.onRender(this.portalElement, this.getTargetElement());
-        }
-      });
+      unstable_renderSubtreeIntoContainer(
```

Comment:
This still works in React 16 - when we upgrade to React 16 and make use of their portals we will be able to delete all this code.